addons/project/views/project_task_views.xml

```diff
@@ -661,7 +667,7 @@
                                 <div class="text-muted d-flex flex-column">
                                     <field t-if="record.parent_id.raw_value" invisible="context.get('default_parent_id', False)" name="parent_id"/>
                                     <field invisible="context.get('default_project_id', False)" name="project_id" options="{'no_open': True}"/>
-                                    <field name="partner_id"/>
+                                    <field name="partner_id" invisible="has_template_ancestor"/>
```

Comment:
is it really needed? 🤔 

Since we don't display the template task in all tasks views (except the ones used in task template view) we could revert that change, no?

TBH, if we keep that we will potentially have a performance issue when there is too many tasks to compute.